docs/source/essentials/server.md

```diff
@@ -159,11 +95,55 @@ And the server should return a simple response:
 ```json
 {
   "data": {
-    "hello": null
+    "hello": "world"
   }
 }
 ```
 
+Your server itself is hosted at http://localhost:4000/graphql. This would be the endpoint you pass to set up Apollo Client.
+
+<h2 id="integrations">Server integrations</h2>
```

Comment:
"Server integrations" => "Server Integrations"

Reply:
http://localhost:4000/graphql should be in code quotes. =>``http://localhost:4000/graphql`.`

Reply:
We usually don't capitalize the second word on the titles since it looks overly formal. For urls, we usually use the link highlighting instead of code quotes.

Reply:
Yeah, we use sentence case for the titles.  There are some additional guidelines worth reviewing in the [docs docs](https://docs-docs.netlify.com/docs/docs/formatting.html#sentence-case)!